lua/cmp/float.lua

```diff
@@ -123,7 +123,7 @@ float.scroll = function(self, delta)
 
     vim.defer_fn(function()
       vim.api.nvim_buf_call(buf, function()
-        vim.cmd('normal! ' .. top .. 'zt')
+        vim.api.nvim_command('normal! ' .. top .. 'zt')
```

Comment:
Here, `vim.api.nvim_command` is faster than `vim.cmd` because `vim.cmd` takes multiple lines into account whereas `vim.api.nvim_command` only works on one. Thus the logic of dealing with multiple lines is absent from `vim.api.nvim_command`. 

You can also personally verify this by running `vim.loop.hrtime` on both statements.